### ISSUE TITLE
Command for scrolling into the page

### ISSUE THREAD

**Issue by MathRobin** [cypress-io/cypress#313]
Hi, thanks for cypress, it's a great tool.

For the moment I use the trick that when click on an element, the page will scroll. Not really the more natural thing. Is it possible to get something like cypress.scrollTo? 

It could be very useful to test parallax and other UX features

**Comment by brian-mann** [comment]
Although we could probably create a more robust solution for this, you can already easily achieve this using native DOM API's.  There is already a `scrollIntoView` method on any `Element`.

```javascript
cy.get("button").then(function($btn){
  $btn.get(0).scrollIntoView()
})
```

We'll keep this issue open because it's come up before and we likely should just add our own cypress command which gives you more options about how the element is scrolled into view.

**Comment by ddeath** [comment]
@brian-mann should it work with `cypress run` ? Can not make the test pass with that code on cli. When running in `cypress open` everything is just fine...

**Comment by RandallKent** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
